app/[locale]/page.tsx

```diff
@@ -1,3 +1,5 @@
+import Slideshow from '@/components/slideshow';
+
```

Comment:
There will be no line break here.

Reply:
unable to commit without the line break
![image](https://github.com/nitkkr-dev/nitkkr/assets/80399932/57070df4-d0b1-4a19-8a1b-458d1a85ecfb)

Reply:
Hmm, then change the below imports to use `@/app/directors-corner` syntax and then you can remove the line break